modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/tasks/DependencyManagementReportRenderer.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.gradle.plugins.target.platform.tasks;
+
+import java.io.PrintWriter;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+
+/**
+ * @author Simon Jiang
+ */
+public class DependencyManagementReportRenderer {
+
+	public DependencyManagementReportRenderer() {
+		this(new PrintWriter(System.out));
```

Comment:
Lets not write to standard out.  Instead lets use standard gradle logging 

getLogger() (see other plugins for how to do this)

logger.lifecycle("output messages to lifecycle) .